### ISSUE TITLE
Multiple Randoms

### ISSUE THREAD

**Issue by NathanBroadway** [Dreadpirate71/DungeonApp#2]
What is having multiple new Random() gain you? Could you just have one? 
Sidenote: Do you know what var means? I don't believe we taught it yet?
[Reference](https://github.com/Dreadpirate71/DungeonApp/blob/main/Dungeon%20App/Combatant.cs#L78)

**Comment by Dreadpirate71** [comment]
Yes one is all that is necessary. When I initially wrote the code I did not fully understand the concept of instantiating an object. The var has not been covered yet. I read about it. I could be wrong but I thought it was used as a generic type so that C# could figure out the proper type based on the expression. Also I looked up the syntax for random number generator and it used var.  So yes don't fully understand var and its proper uses.
